aes/ta/aes_ta.c

```diff
@@ -247,8 +248,8 @@ static TEE_Result set_aes_key(void *session, uint32_t param_types,
 	key_sz = params[0].memref.size;
 
 	if (key_sz != sess->key_size) {
-		EMSG("Worng key size %d, expect %d bytes",
-				key_sz, sess->key_size);
+		EMSG("Wrong key size %" PRIu32 ", expect %d bytes",
```

Comment:
and also for the 2nd `%d` => `" PRIxu32 "`.

Reply:
Of course!